include/bluetooth/services/otc.h

```diff
@@ -3,24 +3,41 @@
  *
  * For use with the Object Transfer Service client (OTC)
  *
- * Copyright (c) 2020 - 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020 - 2022 Nordic Semiconductor ASA
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#ifndef ZEPHYR_INCLUDE_BLUETOOTH_HOST_AUDIO_OTC_H_
-#define ZEPHYR_INCLUDE_BLUETOOTH_HOST_AUDIO_OTC_H_
-
-/* TODO: Temporarily here - clean up, and move alongside the Object Transfer Service */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_OTC_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_OTC_H_
 
 #include <stdbool.h>
 #include <zephyr/types.h>
 #include <bluetooth/gatt.h>
-#include "ots.h"
+#include <bluetooth/services/ots.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define SET_OR_CLEAR_BIT(var, bit_val, set)				\
+	((var) = (set) ? ((var) | bit_val) : ((var) & ~bit_val))
+
+
+/** @brief Date and Time structure.
+ *  TODO: Move somewhere else - bluetooth.h?
+ */
+#define DATE_TIME_FIELD_SIZE 7
+struct bt_date_time {
+	uint16_t year;
+	uint8_t month;
+	uint8_t day;
+	uint8_t hours;
+	uint8_t minutes;
+	uint8_t seconds;
+};
+
+#define BT_OTS_NAME_MAX_SIZE           120
+
 #define BT_OTC_MAX_WRITE_SIZE               23
 #define BT_OTC_UNKNOWN_ID                   0xFF
```

Comment:
Looks like majority of these defines are only used in `ots.c` for internal logic. Could they be made "private" by moving them to `ots.c`?

Reply:
Moved the two last ones.